### PR TITLE
Connect loginsignup

### LINKED ISSUE #75: link between components
- [x] link signup label in Login component to go to SignUp component.

- [x] link login label in SignUp component to go to Login component.

- [x] link Back button to go to previous page

### PR DESCRIPTION
**Pull request by amanshawar**
make the login go to sign up 
and the sign up go login 

fix #75 
related #75

### CODE REVIEW

**Review by orjwan98 — APPROVED** [review]

File: client/src/components/Login/index.js [hunk 1]
```diff
@@ -37,49 +37,54 @@ class Login extends Component {
...
+        />
 
-          <StyledP>
-            <StyledText>First time around here? </StyledText>
-            <StyledSignUp> Sign up</StyledSignUp>
-          </StyledP>
-        </StyledForm>
+        <StyledP>
+          <StyledText>First time around here? </StyledText>
+          <StyledSignUp onClick={this.goSignUp}> Sign up</StyledSignUp>
```
Comment (jema28) [inline client/src/components/Login/index.js:76]: This is in the login page, it should be called StyledLogin - you can extend the StyledSign in the style page but call it StyledLogin.

File: client/src/components/SignUp/signup.style.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ export const StyledLabel = styled.label`
 `;
 
 
-export const StyledSignUp = styled.label`
+export const StyledLogin = styled.label`
```
Comment (jema28) [inline client/src/components/SignUp/signup.style.js:35]: This is the signup styles? Look at comment above. You can extend like this:

`export const StyledSignUp = styled(StyledLogin)`

File: client/src/components/Login/index.js [hunk 3]
```diff
@@ -37,49 +37,54 @@ class Login extends Component {
...
-          <Button />
-          <GButton title="Login" onClick={this.login} />
-          <Footer />
-        </StyledBottom>
-      </React.Fragment>
-    );
-  }
+      <StyledBottom>
+        <Button />
+        <GButton title="Login" onClick={this.login} />
```
Comment (jema28) [inline client/src/components/Login/index.js]: This needs to be in capitals.

**Review by jema28 — CHANGES_REQUESTED** [review]
Awesome job 👍  see comments below.

File: client/src/components/Login/index.js (hunk 3 above)
Comment (amanshawar) [reply client/src/components/Login/index.js]: done

**Review by jema28 — APPROVED** [review]

**Review by FidaaHersh — APPROVED** [review]

**Review by FidaaHersh — APPROVED** [review]

**Pull request merged**